<html>
<head>
<title>TestNG:  Test</title>
<link href="../testng.css" rel="stylesheet" type="text/css" />
<link href="../my-testng.css" rel="stylesheet" type="text/css" />

<style type="text/css">
.log { display: none;} 
.stack-trace { display: none;} 
</style>
<script type="text/javascript">
<!--
function flip(e) {
  current = e.style.display;
  if (current == 'block') {
    e.style.display = 'none';
    return 0;
  }
  else {
    e.style.display = 'block';
    return 1;
  }
}

function toggleBox(szDivId, elem, msg1, msg2)
{
  var res = -1;  if (document.getElementById) {
    res = flip(document.getElementById(szDivId));
  }
  else if (document.all) {
    // this is the way old msie versions work
    res = flip(document.all[szDivId]);
  }
  if(elem) {
    if(res == 0) elem.innerHTML = msg1; else elem.innerHTML = msg2;
  }

}

function toggleAllBoxes() {
  if (document.getElementsByTagName) {
    d = document.getElementsByTagName('div');
    for (i = 0; i < d.length; i++) {
      if (d[i].className == 'log') {
        flip(d[i]);
      }
    }
  }
}

// -->
</script>

</head>
<body>
<h2 align='center'>Test</h2><table border='1' align="center">
<tr>
<td>Tests passed/Failed/Skipped:</td><td>12/5/0</td>
</tr><tr>
<td>Started on:</td><td>Thu Jul 25 11:19:42 IST 2024</td>
</tr>
<tr><td>Total time:</td><td>73 seconds (73555 ms)</td>
</tr><tr>
<td>Included groups:</td><td></td>
</tr><tr>
<td>Excluded groups:</td><td></td>
</tr>
</table><p/>
<small><i>(Hover the method name to see the test class name)</i></small><p/>
<table width='100%' border='1' class='invocation-failed'>
<tr><td colspan='4' align='center'><b>FAILED TESTS</b></td></tr>
<tr><td><b>Test method</b></td>
<td><b>Attribute(s)</b></td>
<td width="30%"><b>Exception</b></td>
<td width="10%"><b>Time (seconds)</b></td>
<td><b>Instance</b></td>
</tr>
<tr>
<td title='testCases.TC_005_RegisterDataDrivenTest.validateRegistrationWithDiffCredentials()'><b>validateRegistrationWithDiffCredentials</b><br>Test class: testCases.TC_005_RegisterDataDrivenTest<br>Parameters: Sam, Kold, sam@outlook.com, 263562323, sam@123, sam@123, Valid</td>
<td><div><pre>java.lang.AssertionError: expected [true] but found [false]
	at testCases.TC_005_RegisterDataDrivenTest.validateRegistrationWithDiffCredentials(TC_005_RegisterDataDrivenTest.java:60)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 31 stack frames</pre></div><a href='#' onClick='toggleBox("stack-trace1985569416", this, "Click to show all stack frames", "Click to hide stack frames")'>Click to show all stack frames</a>
<div class='stack-trace' id='stack-trace1985569416'><pre>java.lang.AssertionError: expected [true] but found [false]
	at org.testng.Assert.fail(Assert.java:111)
	at org.testng.Assert.failNotEquals(Assert.java:1578)
	at org.testng.Assert.assertTrue(Assert.java:57)
	at org.testng.Assert.assertTrue(Assert.java:67)
	at testCases.TC_005_RegisterDataDrivenTest.validateRegistrationWithDiffCredentials(TC_005_RegisterDataDrivenTest.java:60)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)
	at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:686)
	at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:230)
	at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)
	at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:992)
	at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)
	at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)
	at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
	at org.testng.TestRunner.privateRun(TestRunner.java:739)
	at org.testng.TestRunner.run(TestRunner.java:614)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:421)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:413)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:373)
	at org.testng.SuiteRunner.run(SuiteRunner.java:312)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1274)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1208)
	at org.testng.TestNG.runSuites(TestNG.java:1112)
	at org.testng.TestNG.run(TestNG.java:1079)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
</pre></div></td>
<td>1</td>
<td>testCases.TC_005_RegisterDataDrivenTest@1b2c4efb</td></tr>
<tr>
<td title='testCases.TC_004_LoginDataDrivenTests.validateLoginWithDiffCredentials()'><b>validateLoginWithDiffCredentials</b><br>Test class: testCases.TC_004_LoginDataDrivenTests<br>Parameters: laksh@yahoo.com, Lakshmi, Invalid</td>
<td><div><pre>java.lang.AssertionError: expected [true] but found [false]
	at testCases.TC_004_LoginDataDrivenTests.validateLoginWithDiffCredentials(TC_004_LoginDataDrivenTests.java:63)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 31 stack frames</pre></div><a href='#' onClick='toggleBox("stack-trace1122960426", this, "Click to show all stack frames", "Click to hide stack frames")'>Click to show all stack frames</a>
<div class='stack-trace' id='stack-trace1122960426'><pre>java.lang.AssertionError: expected [true] but found [false]
	at org.testng.Assert.fail(Assert.java:111)
	at org.testng.Assert.failNotEquals(Assert.java:1578)
	at org.testng.Assert.assertTrue(Assert.java:57)
	at org.testng.Assert.assertTrue(Assert.java:67)
	at testCases.TC_004_LoginDataDrivenTests.validateLoginWithDiffCredentials(TC_004_LoginDataDrivenTests.java:63)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)
	at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:686)
	at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:230)
	at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)
	at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:992)
	at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)
	at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)
	at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
	at org.testng.TestRunner.privateRun(TestRunner.java:739)
	at org.testng.TestRunner.run(TestRunner.java:614)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:421)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:413)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:373)
	at org.testng.SuiteRunner.run(SuiteRunner.java:312)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1274)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1208)
	at org.testng.TestNG.runSuites(TestNG.java:1112)
	at org.testng.TestNG.run(TestNG.java:1079)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
</pre></div></td>
<td>1</td>
<td>testCases.TC_004_LoginDataDrivenTests@4bff64c2</td></tr>
<tr>
<td title='testCases.TC_004_LoginDataDrivenTests.validateLoginWithDiffCredentials()'><b>validateLoginWithDiffCredentials</b><br>Test class: testCases.TC_004_LoginDataDrivenTests<br>Parameters: pavanoltraining@gmail.com, test@123, Valid</td>
<td><div><pre>java.lang.AssertionError: expected [true] but found [false]
	at testCases.TC_004_LoginDataDrivenTests.validateLoginWithDiffCredentials(TC_004_LoginDataDrivenTests.java:54)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 31 stack frames</pre></div><a href='#' onClick='toggleBox("stack-trace399683701", this, "Click to show all stack frames", "Click to hide stack frames")'>Click to show all stack frames</a>
<div class='stack-trace' id='stack-trace399683701'><pre>java.lang.AssertionError: expected [true] but found [false]
	at org.testng.Assert.fail(Assert.java:111)
	at org.testng.Assert.failNotEquals(Assert.java:1578)
	at org.testng.Assert.assertTrue(Assert.java:57)
	at org.testng.Assert.assertTrue(Assert.java:67)
	at testCases.TC_004_LoginDataDrivenTests.validateLoginWithDiffCredentials(TC_004_LoginDataDrivenTests.java:54)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)
	at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:686)
	at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:230)
	at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)
	at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:992)
	at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)
	at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)
	at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
	at org.testng.TestRunner.privateRun(TestRunner.java:739)
	at org.testng.TestRunner.run(TestRunner.java:614)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:421)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:413)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:373)
	at org.testng.SuiteRunner.run(SuiteRunner.java:312)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1274)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1208)
	at org.testng.TestNG.runSuites(TestNG.java:1112)
	at org.testng.TestNG.run(TestNG.java:1079)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
</pre></div></td>
<td>6</td>
<td>testCases.TC_004_LoginDataDrivenTests@4bff64c2</td></tr>
<tr>
<td title='testCases.TC_004_LoginDataDrivenTests.validateLoginWithDiffCredentials()'><b>validateLoginWithDiffCredentials</b><br>Test class: testCases.TC_004_LoginDataDrivenTests<br>Parameters: lakshmi@yahoo.com, Laxmi, Invalid</td>
<td><div><pre>java.lang.AssertionError: expected [true] but found [false]
	at testCases.TC_004_LoginDataDrivenTests.validateLoginWithDiffCredentials(TC_004_LoginDataDrivenTests.java:63)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 31 stack frames</pre></div><a href='#' onClick='toggleBox("stack-trace1947910788", this, "Click to show all stack frames", "Click to hide stack frames")'>Click to show all stack frames</a>
<div class='stack-trace' id='stack-trace1947910788'><pre>java.lang.AssertionError: expected [true] but found [false]
	at org.testng.Assert.fail(Assert.java:111)
	at org.testng.Assert.failNotEquals(Assert.java:1578)
	at org.testng.Assert.assertTrue(Assert.java:57)
	at org.testng.Assert.assertTrue(Assert.java:67)
	at testCases.TC_004_LoginDataDrivenTests.validateLoginWithDiffCredentials(TC_004_LoginDataDrivenTests.java:63)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)
	at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:686)
	at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:230)
	at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)
	at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:992)
	at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)
	at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)
	at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
	at org.testng.TestRunner.privateRun(TestRunner.java:739)
	at org.testng.TestRunner.run(TestRunner.java:614)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:421)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:413)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:373)
	at org.testng.SuiteRunner.run(SuiteRunner.java:312)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1274)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1208)
	at org.testng.TestNG.runSuites(TestNG.java:1112)
	at org.testng.TestNG.run(TestNG.java:1079)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
</pre></div></td>
<td>1</td>
<td>testCases.TC_004_LoginDataDrivenTests@4bff64c2</td></tr>
<tr>
<td title='testCases.TC_005_RegisterDataDrivenTest.validateRegistrationWithDiffCredentials()'><b>validateRegistrationWithDiffCredentials</b><br>Test class: testCases.TC_005_RegisterDataDrivenTest<br>Parameters: Hanson, Lee, zzz989@gmail.com, 444456789, zzz989@123, zzz989@123, Valid</td>
<td><div><pre>java.lang.AssertionError: expected [true] but found [false]
	at testCases.TC_005_RegisterDataDrivenTest.validateRegistrationWithDiffCredentials(TC_005_RegisterDataDrivenTest.java:60)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 31 stack frames</pre></div><a href='#' onClick='toggleBox("stack-trace391479787", this, "Click to show all stack frames", "Click to hide stack frames")'>Click to show all stack frames</a>
<div class='stack-trace' id='stack-trace391479787'><pre>java.lang.AssertionError: expected [true] but found [false]
	at org.testng.Assert.fail(Assert.java:111)
	at org.testng.Assert.failNotEquals(Assert.java:1578)
	at org.testng.Assert.assertTrue(Assert.java:57)
	at org.testng.Assert.assertTrue(Assert.java:67)
	at testCases.TC_005_RegisterDataDrivenTest.validateRegistrationWithDiffCredentials(TC_005_RegisterDataDrivenTest.java:60)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)
	at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:686)
	at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:230)
	at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)
	at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:992)
	at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)
	at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)
	at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
	at org.testng.TestRunner.privateRun(TestRunner.java:739)
	at org.testng.TestRunner.run(TestRunner.java:614)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:421)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:413)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:373)
	at org.testng.SuiteRunner.run(SuiteRunner.java:312)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1274)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1208)
	at org.testng.TestNG.runSuites(TestNG.java:1112)
	at org.testng.TestNG.run(TestNG.java:1079)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
</pre></div></td>
<td>1</td>
<td>testCases.TC_005_RegisterDataDrivenTest@1b2c4efb</td></tr>
</table><p>
<table width='100%' border='1' class='invocation-passed'>
<tr><td colspan='4' align='center'><b>PASSED TESTS</b></td></tr>
<tr><td><b>Test method</b></td>
<td><b>Attribute(s)</b></td>
<td width="30%"><b>Exception</b></td>
<td width="10%"><b>Time (seconds)</b></td>
<td><b>Instance</b></td>
</tr>
<tr>
<td title='testCases.TC_005_RegisterDataDrivenTest.validateRegistrationWithDiffCredentials()'><b>validateRegistrationWithDiffCredentials</b><br>Test class: testCases.TC_005_RegisterDataDrivenTest<br>Parameters: Ken, Japher, ken1993@gmail.com, 536253562, ken@123, ken@123, Invalid</td>
<td></td>
<td>0</td>
<td>testCases.TC_005_RegisterDataDrivenTest@1b2c4efb</td></tr>
<tr>
<td title='testCases.TC_004_LoginDataDrivenTests.validateLoginWithDiffCredentials()'><b>validateLoginWithDiffCredentials</b><br>Test class: testCases.TC_004_LoginDataDrivenTests<br>Parameters: silco8918@gmail.com, HelloWorld@123, Valid</td>
<td></td>
<td>1</td>
<td>testCases.TC_004_LoginDataDrivenTests@4bff64c2</td></tr>
<tr>
<td title='testCases.TC_004_LoginDataDrivenTests.validateLoginWithDiffCredentials()'><b>validateLoginWithDiffCredentials</b><br>Test class: testCases.TC_004_LoginDataDrivenTests<br>Parameters: laks@yahoo.com, xyz, Invalid</td>
<td></td>
<td>5</td>
<td>testCases.TC_004_LoginDataDrivenTests@4bff64c2</td></tr>
<tr>
<td title='testCases.TC_005_RegisterDataDrivenTest.validateRegistrationWithDiffCredentials()'><b>validateRegistrationWithDiffCredentials</b><br>Test class: testCases.TC_005_RegisterDataDrivenTest<br>Parameters: Merry, Curie, merry123@yahoo.com, 4444363522, merryC@123, merryC@123, Invalid</td>
<td></td>
<td>1</td>
<td>testCases.TC_005_RegisterDataDrivenTest@1b2c4efb</td></tr>
<tr>
<td title='testCases.TC_002_NavigateToRegister.testNavigateToRegisterViaLogin_Register()'><b>testNavigateToRegisterViaLogin_Register</b><br>Test class: testCases.TC_002_NavigateToRegister</td>
<td></td>
<td>3</td>
<td>testCases.TC_002_NavigateToRegister@5829e4f4</td></tr>
<tr>
<td title='testCases.TC_003_ValidLogin.testValidLogin()'><b>testValidLogin</b><br>Test class: testCases.TC_003_ValidLogin</td>
<td></td>
<td>1</td>
<td>testCases.TC_003_ValidLogin@4218500f</td></tr>
<tr>
<td title='testCases.TC_002_NavigateToRegister.testNavigateToRegisterViaLogin_NewCustomer()'><b>testNavigateToRegisterViaLogin_NewCustomer</b><br>Test class: testCases.TC_002_NavigateToRegister</td>
<td></td>
<td>3</td>
<td>testCases.TC_002_NavigateToRegister@5829e4f4</td></tr>
<tr>
<td title='testCases.TC_005_RegisterDataDrivenTest.validateRegistrationWithDiffCredentials()'><b>validateRegistrationWithDiffCredentials</b><br>Test class: testCases.TC_005_RegisterDataDrivenTest<br>Parameters: Klue, xyz, klue262@gmail.com, 444464464, klue@123, klue@123, Invalid</td>
<td></td>
<td>1</td>
<td>testCases.TC_005_RegisterDataDrivenTest@1b2c4efb</td></tr>
<tr>
<td title='testCases.TC_004_LoginDataDrivenTests.validateLoginWithDiffCredentials()'><b>validateLoginWithDiffCredentials</b><br>Test class: testCases.TC_004_LoginDataDrivenTests<br>Parameters: abc123@gmail.com, test@123, Valid</td>
<td></td>
<td>1</td>
<td>testCases.TC_004_LoginDataDrivenTests@4bff64c2</td></tr>
<tr>
<td title='testCases.TC_004_LoginDataDrivenTests.validateLoginWithDiffCredentials()'><b>validateLoginWithDiffCredentials</b><br>Test class: testCases.TC_004_LoginDataDrivenTests<br>Parameters: , HelloWorld@123, Invalid</td>
<td></td>
<td>5</td>
<td>testCases.TC_004_LoginDataDrivenTests@4bff64c2</td></tr>
<tr>
<td title='testCases.TC_001_AccountRegistrationTest.registrationTest()'><b>registrationTest</b><br>Test class: testCases.TC_001_AccountRegistrationTest</td>
<td></td>
<td>4</td>
<td>testCases.TC_001_AccountRegistrationTest@5c08c46a</td></tr>
<tr>
<td title='testCases.TC_002_NavigateToRegister.testNavigateToRegisterViaRegister()'><b>testNavigateToRegisterViaRegister</b><br>Test class: testCases.TC_002_NavigateToRegister</td>
<td></td>
<td>0</td>
<td>testCases.TC_002_NavigateToRegister@5829e4f4</td></tr>
</table><p>
</body>
</html>